technical/20171122_tomesh-tech-status-update.md

```diff
@@ -0,0 +1,107 @@
+---
+title: tomesh Tech Status Update
+location: Toronto
+date: 2017-11-22
+---
+
+# Overview
+
+This document is a general overview of the technical pieces associate with Toronto Mesh (tomesh). It is intended for those trying to learn more about tomesh, presenting at a high level the pieces that we hope one day will fit together into a community mesh network, and the current statuses on these initiatives.
+
+A **node** is a bundle of free and open-source software running on off-the-shelf and reproducible low-cost hardware. These nodes, each one owned and operated by a user who **peers** with neighbouring nodes, form the **mesh network**. This network is analogous to our often privately-owned last-mile networks, except it is maintained by the community of users. We use the network for private and secure communication, including real-time use cases (e.g. instant messaging, voice and video chats) and hosting persistent services (e.g. a blog, a web application, peer-to-peer services).
+
+![Nodes](../images/20171122_tomesh-tech-status-update.jpg?raw=true)
+
+# Design Goals
+
+ * protocol-level guarantees of private and secure communication among nodes
```

Comment:
_protocol-level guarantees_ seems a little ambiguous in this and the following point. Would removing it be descriptive enough for the two points?